examples/tensorflow_converter/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required( VERSION 3.2 )
+
+project( tensorflow_converter )
+
+find_package( Protobuf 3.0.0 REQUIRED )
+
+file(GLOB PROTO_FILES ${CMAKE_CURRENT_SOURCE_DIR}/../../tiny_dnn/io/tensorflow/tensorflow/core/framework/*.proto)
+
+# generate CPP file for connexion between protobuf and tiny_dnn 
+PROTOBUF_GENERATE_CPP( PROTO_SRC PROTO_HDR ${PROTO_FILES} )
```

Comment:
@rperrot I adapted your caffe_converter cmake for a tensorflow parser that we developed during the GSOC. However, with `file(GLOB ...)` I'm not able to compile multiple proto files at once. Any idea?